autoware_launch/launch/autoware.launch.xml

```diff
@@ -49,6 +49,7 @@
     <!-- options for mode: camera_lidar_fusion, lidar, camera -->
     <arg name="mode" value="camera_lidar_fusion" />
     <arg name="vehicle_param_file" value="$(find-pkg-share $(var vehicle_model)_description)/config/vehicle_info.param.yaml"/>
+    <arg name="pointcloud_map_path" value="$(var map_path)/$(var pointcloud_map_file)"/>
```

Comment:
@yukkysaito In current implementaiton, map_path is required by perception_launch, but it's not nice. I have some idea to avoid this, please explain about that next week.

Reply:
(We talked about this today.)

Reply:
I fixed this issue by publishing pointcloud map hash from map_hash_generator and use it in elevation map loader.
 - https://github.com/tier4/AutowareArchitectureProposal.iv/pull/745
 - https://github.com/tier4/AutowareArchitectureProposal.iv/pull/740